src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-policy-guestconfig/main.combined.bicep

```diff
@@ -0,0 +1,29 @@
+﻿// $1 = 'name'

```

Comment:
combined test templates should not contain comments or placeholders. e.g. https://github.com/Azure/bicep/blob/10b1ccb60dbdbc0bcba8e908e66fa30cfd029c51/src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-aks-cluster/main.combined.bicep